src/main/java/net/comments/objects/CommentErrors.java

```diff
@@ -0,0 +1,68 @@
+package net.comments.objects;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.util.NoSuchElementException;
+
+public class CommentErrors {
```

Comment:
@oteplova It's a bad idea to hardcode messages. Try to create a method which will return an error message if present, otherwise empty message. You have to rework this implementation.